stylesheets/styles.css

```diff
@@ -0,0 +1,621 @@
+@font-face {
+  font-family: "Cocogoose Letterpress";
+  src: url(../../fonts/COCOGOOSELETTERPRESS_TRIAL.ttf);
+}
+
+@font-face {
+  font-family: "Lato";
+  src: url("https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap");
+}
```

Comment:
- [ ] Thank you for implementing all my previous changes however you haven't correctly fixed the fonts that I mentioned before. currently if you inspect your page you'll notice that the only font being used is Arial. 

to clarify a little more the problem seems to occur from sections like this. you don't want to create a font in this manner if the URL is not a link to an exact font like is the case from the Lato font link you got from google fonts. use `@import` here and it will automatically handle the `@font-face` rule for you correctly. what you want to do is remove this `@font-face` rules and simply include a `@import` rule. 

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap');
```